cookbooks/tile/attributes/default.rb

```diff
@@ -1,6 +1,6 @@
 default[:tile][:database][:cluster] = "14/main"
 default[:tile][:database][:postgis] = "3"
-default[:tile][:database][:node_file] = "/store/database/nodes"
+default[:tile][:database][:nodes_store] = :flat
```

Comment:
Why is this changing the `tile` cookbook?

Reply:
Whoops - that's a mistake. I must have mixed up the many default.rb windows I had open.